README.md

```diff
@@ -52,8 +52,8 @@ git clone https://github.com/ethereum/remix-project.git
 cd remix-project
 npm install
 npm run build:libs // Build remix libs
-nx build
-nx serve
+npm run build
+npm run serve
```

Comment:
nx build remix-ide
nx serve

should work

Reply:
sure it might work but isn't it better to use the local binaries and run the command from the script instead of requiring to download an external global library? It just seems like if the script is there we might as well use it... seems like a better practice to me